interfaces/builtin/u2f_devices.go

```diff
@@ -102,9 +102,14 @@ var u2fDevices = []u2fDevice{
 		ProductIDPattern: "5026",
 	},
 	{
-		Name:             "Tomu board + chopstx U2F",
+		Name:             "Tomu board + chopstx U2F + SoloKeys",
 		VendorIDPattern:  "0483",
-		ProductIDPattern: "cdab",
+		ProductIDPattern: "cdab|a2ca",
+	},
+	{
+		Name:             "SoloKeys",
+		VendorIDPattern:  "1209",
+		ProductIDPattern: "5070|50b0",
```

Comment:
Note to other reviewers, this VID is a "generic" VID that has since been adopted by the http://pid.codes project and the SoloKeys devices are referenced there: http://pid.codes/1209/ but these devices are not referenced however in the http://www.linux-usb.org/usb.ids page